tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -73,32 +74,44 @@ Upload subject to new release
     [Tags]  HappyPath
     user clicks link  Data and files
     user waits until page contains element  id:dataFileUploadForm-subjectTitle
-    user enters text into element  id:dataFileUploadForm-subjectTitle   UI test subject
-    user chooses file   id:dataFileUploadForm-dataFile       ${CURDIR}${/}files${/}upload-file-test.csv
-    user chooses file   id:dataFileUploadForm-metadataFile   ${CURDIR}${/}files${/}upload-file-test.meta.csv
+    user enters text into element  id:dataFileUploadForm-subjectTitle   ${SUBJECT_NAME}
+    user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}upload-file-test.csv
+    user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}upload-file-test.meta.csv
     user clicks button  Upload data files
 
     user waits until h2 is visible  Uploaded data files
-    user waits until page contains accordion section   UI test subject
-    user opens accordion section   UI test subject
+    user waits until page contains accordion section   ${SUBJECT_NAME}
+    user opens accordion section   ${SUBJECT_NAME}
 
-    ${section}=  user gets accordion section content element  UI test subject
-    user checks headed table body row contains  Subject title    UI test subject  ${section}
+    ${section}=  user gets accordion section content element  ${SUBJECT_NAME}
+    user checks headed table body row contains  Subject title    ${SUBJECT_NAME}  ${section}
     user checks headed table body row contains  Data file        upload-file-test.csv  ${section}
     user checks headed table body row contains  Metadata file    upload-file-test.meta.csv  ${section}
     user checks headed table body row contains  Number of rows   159  ${section}
     user checks headed table body row contains  Data file size   15 Kb  ${section}
     user checks headed table body row contains  Status           Complete  ${section}  360
 
+Add meta guidance to subject
+    [Tags]  HappyPath
+    user clicks link  Metadata guidance
+    user waits until h2 is visible  Public metadata guidance document
+
+    user waits until page contains accordion section  ${SUBJECT_NAME}
+    user opens accordion section  ${SUBJECT_NAME}
+    ${text_element}=  user gets meta guidance accordion text element  ${SUBJECT_NAME}
```

Comment:
Given the suggest keyword change, this would be something like `${editor}` instead